src/plugins/audioPlugin.js

```diff
@@ -20,15 +31,104 @@ export default function createAudioPlugin() {
     };
   };
 
-  const embedToHTML = embed => {
-    const { audio: { title, audioFile: { mimeType, url } } } = embed;
-    return `<figure class="article_audio"><audio controls type="${mimeType}" src="${url}"></audio><figcaption>${
-      title.title
-    }</figcaption></figure>`;
+  const onError = ({ audio }, locale) => {
+    const { audioFile: { mimeType, url } = {} } = audio || {};
+    return renderToStaticMarkup(
+      <Figure>
+        {audio ? (
+          <AudioPlayer type={mimeType} src={url} />
+        ) : (
+          <AudioPlayer disabled />
+        )}
+        <figcaption>{t(locale, 'audio.error.caption')}</figcaption>
+      </Figure>
+    );
+  };
+
+  const AudioActionButtons = ({ locale, src, copyString }) => [
+    <Button
+      key="copy"
+      outline
+      data-copied-title={t(locale, 'reference.copied')}
+      data-copy-string={copyString}>
+      {t(locale, 'reference.copy')}
+    </Button>,
+    <a
+      key="download"
+      href={src}
+      className="c-button c-button--outline"
+      download>
+      {t(locale, 'audio.download')}
+    </a>,
+  ];
+
+  AudioActionButtons.propTypes = {
+    locale: PropTypes.string.isRequired,
+    src: PropTypes.string.isRequired,
+    copyString: PropTypes.string.isRequired,
+  };
+
+  const embedToHTML = ({ audio }, locale) => {
+    const {
+      id,
+      title: { title },
+      audioFile: { mimeType, url },
+      copyright: {
+        creators,
+        license: { license: licenseAbbreviation },
+        origin,
+      },
+    } = audio;
+
+    const license = getLicenseByAbbreviation(licenseAbbreviation, locale);
+
+    const contributors = getGroupedContributorDescriptionList(
+      audio.copyright,
+      locale
+    ).map(item => ({
+      name: item.description,
+      type: item.label,
+    }));
+
+    const figureLicenseDialogId = `audio-${id}`;
+    const messages = {
+      title: t(locale, 'title'),
+      close: t(locale, 'close'),
+      rulesForUse: t(locale, 'audio.rulesForUse'),
+      learnAboutLicenses: t(locale, 'learnAboutLicenses'),
+      source: t(locale, 'source'),
+    };
+    return renderToStaticMarkup(
+      <Figure>
+        <AudioPlayer type={mimeType} src={url} />
```

Comment:
Send gjerne inn title som prop til AudioPlayer også. Det er strengt tatt caption vi mangler for lydfiler (fra api'et), men greit å bruke title der også.

Reply:
Ok, jeg syns ikke det såg noe bra ut med overskrift det samme som caption, men da bruker jeg begge enn så lenge.